src/main/java/save/Id.java

```diff
@@ -0,0 +1,40 @@
+package save;
+
+public enum Id {
+    A {
+        public String toString() {
```

Comment:
Recommend to use the builtin toString and valueOf methods
that way you don't have to use 1 letter ids and can use more descriptive ones